resources/template/menu.php

```diff
@@ -30,6 +30,7 @@ public function render_header()
 
         echo " <input type='checkbox' id='check'>
             <!--header area start-->
+            <meta name='viewport' content='width=device-width, initial-scale=1'>
```

Comment:
Its better to keep `<meta>` tag inside `<head>`, and I have already added it wherever it as missing. You can remove it from here.

Reply:
oo right i forgot
about it